Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -607,10 +607,13 @@ private async void CreateControllerModelFromPlatformSDK(uint interactionSourceId
                 }
             }
 
-            if (gltfGameObject == null)
+            if (gltfGameObject != null)
+            {
+                isControllerModelLoaded = true;
```

Comment:
Should we assign `isControllerModelLoaded` in any case? 
i.e. 
`isControllerModelLoaded = (gltfGameObject != null);`

Otherwise, if `CreateControllerModelFromPlatformSDK` get called a second time the flag may not be set to the correct value if loading fails.

Reply:
good point! I'll change that :)